src/VisualStudio/CSharp/Impl/VSPackage.resx

```diff
@@ -142,31 +142,161 @@
     <comment>"C#" node help text in profile Import/Export.</comment>
   </data>
   <data name="306" xml:space="preserve">
-    <value>Highlight references to symbol under cursor;Highlight related keywords under cursor;Enter outlining mode when files open;Show procedure line separators;Generate XML documentation comments;Show diagnostics for closed files;Show preview for rename tracking;Place 'System' directives first when sorting usings;Don't put ref or out on custom structs;Regex;Colorize regular expressions;Highlight related components under cursor;Report invalid regular expressions</value>
+    <value>Show diagnostics for closed files;
```

Comment:
❓ Can we create a test that verifies the text of each text control in the options page control appears in this resource?

❓ Can we create a test that verifies each substring (separated by `;`) of this resource appears as the text of a control?

Reply:
I would be fine creating an issue on creating test on this and check this in. product is broken now and there was no test before. so no need to be blocked until product is fixed before we figure out how to test this feature.